runtime/pom.xml

```diff
@@ -173,6 +173,12 @@
             <version>6.0.53</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>${logback.classic.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this needed? I believe it was removed because of conflicts in NSX:  https://github.com/CorfuDB/CorfuDB/pull/2841/.

Reply:
It is for inspecting the logs in unit test of `ClientHandshakeHandler`. I set the scope to be test hoping to avoid the issue, but I will double check.

Reply:
As we no longer inspect log message in test cases, it is now removed, thanks!